tests/test_run_rvic.py

```diff
@@ -17,16 +27,26 @@
                 "grid_id": "COLUMBIA",
                 "run_startdate": "2012-12-01-00",
                 "stop_date": "2012-12-31",
-                "pour_points": "/home/eyvorchuk/Documents/osprey/tests/data/samples/sample_pour.txt",
-                "uh_box": "/home/eyvorchuk/Documents/osprey/tests/data/samples/uhbox.csv",
-                "routing": "/home/eyvorchuk/Documents/osprey/tests/data/samples/sample_flow_parameters.nc",
-                "domain": "/home/eyvorchuk/Documents/osprey/tests/data/samples/sample_routing_domain.nc",
+                "pour_points": resource_filename(
+                    __name__, "data/samples/sample_pour.txt"
```

Comment:
Instead of `__name__` could we use the package name? I think it would be `"test"` in this case.